src/charm.py

```diff
@@ -62,6 +64,8 @@
 GATEWAY_API_CRDS_LABEL = "gateway-apis-crds"
 GATEWAY_API_CRDS_RESOURCE_TYPES = {CustomResourceDefinition}
 
+METRICS_LABELS = {"charms.canonical.com/istio.io.metrics": "aggregated"}
```

Comment:
Do we want to include the app name and model uuid here? While it's unlikely we will support two instances of istio on the same mesh, making the label unique to this app could leave that option open for the future.